guide/how-it-works/lightning-services.md

```diff
@@ -0,0 +1,330 @@
+---
+layout: guide
+title: Lightning Services
+description: Third party services that solve common user experience problems when connecting to and using the lightning network.
+nav_order: 4
+parent: How it works
+permalink: /guide/how-it-works/lightning-services/
+redirect_from:
+ - /guide/payments/send/lightning-service-providers/
+main_classes: -no-top-padding
+---
+
+<!--
+
+Editor's notes
+
+Illustration sources: https://www.figma.com/file/qr4P17z6WSPADm6oW0cKw2/?node-id=701%3A5911
+
+-->
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/lightning-services/lightning-services.jpg"
+   retina = "/assets/images/guide/how-it-works/lightning-services/lightning-services@2x.jpg"
+   mobile = "/assets/images/guide/how-it-works/lightning-services/lightning-services-mobile.jpg"
+   mobileRetina = "/assets/images/guide/how-it-works/lightning-services/lightning-services-mobile@2x.jpg"
+   alt-text = "Lightning service providers header image."
```

Comment:
Alt-text is not useful. If the image does not provide important information for the reader, you can leave the alt-text blank.